.travis.yml

```diff
@@ -6,7 +6,7 @@ services:
   - postgresql
 
 addons:
-  postgresql: "9.3"
+  postgresql: "9.4"
```

Comment:
Did we move production up to 9.4?

Reply:
yes

Reply:
to accommodate jsonb column types

Reply:
Ah - no - we did not. 
The database for `travis-yml-checker` uses 9.4 in order to accommodate `jsonb` column types.

Let's do an update next week then...